client/context.go

```diff
@@ -79,6 +79,10 @@ type Context struct {
 	AddressCodec          address.Codec
 	ValidatorAddressCodec address.Codec
 	ConsensusAddressCodec address.Codec
+
+	// Bech32 address prefixes.
```

Comment:
We only need this for the keyring right? Is there a way to not need this? I am not sure how, and it is fine to keep if not, but curious if we can avoid it and that basically you only need to define things only once.

Reply:
This is actually needed to create keys with ledger and to parse hex keys into bech32 ones. The keyring does not make use of this.
Can't find a way to avoid it.

Reply:
Alright, too bad 👍